central/audit/audit.go

```diff
@@ -152,6 +153,15 @@ func (a *audit) UnaryServerInterceptor() func(ctx context.Context, req interface
 	}
 }
 
+func (a *audit) HTTPInterceptor(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		statusTrackingWriter := httputil.NewStatusTrackingWriter(w)
+		handler.ServeHTTP(statusTrackingWriter, r)
+
+		go a.SendAdhocAuditMessage(r.Context(), r, r.RequestURI, interceptor.AuthStatus{}, statusTrackingWriter.GetStatusCodeError())
```

Comment:
1. I figured since `SendAdhocAuditMessage` is identical to `sendAuditMessage`, why not use the latter?
2. Since they are identical, perhaps we can refactor these `SendAdhocAuditMessage` to call `sendAuditMessage`?

Reply:
Nice catch! I didn't notice it.

I did some bigger refactoring here:
1. removed `sendAuditMessage`
2. renamed `SendAdhocAuditMessage` to `SendAuditMessage`
3. set internal function call to use `SendAuditMessage`